backend/apps/objects/views.py

```
@@ -142,19 +153,21 @@ def post(self, request):
-                tags=body.get('tags', ''),
+                description=body.get("description", ""),
+                location=body.get("location", ""),
+                image_url=body.get("image_url", None),
+                tags=body.get("tags", ""),
                 residence=request.residence,
             )
-            return JsonResponse({'id': obj.id, 'detail': 'Object created successfully'}, status=201)
+            return JsonResponse(
+                {"id": obj.id, "detail": "Object created successfully"}, status=201
+            )
         except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)